breakpoints.js

```diff
@@ -27,6 +29,32 @@
 		lastSize = 0;
 	};
 	
+		var scrollbarWidth = 0;
+
+		$.getScrollbarWidth = function() {
+		/* Gets the width of the OS scrollbar		
+		https://github.com/brandonaaron/jquery-getscrollbarwidth/blob/master/jquery.getscrollbarwidth.js */
+		
+			if ( !scrollbarWidth ) {
+				if ( $.browser.msie ) {
+					var $textarea1 = $('<textarea cols="10" rows="2"></textarea>')
+					.css({ position: 'absolute', top: -1000, left: -1000 }).appendTo('body'),
+					$textarea2 = $('<textarea cols="10" rows="2" style="overflow: hidden;"></textarea>')
+					.css({ position: 'absolute', top: -1000, left: -1000 }).appendTo('body');
+					scrollbarWidth = $textarea1.width() - $textarea2.width();
+					$textarea1.add($textarea2).remove();
+				} else {
+					var $div = $('<div />')
+					.css({ width: 100, height: 100, overflow: 'auto', position: 'absolute', top: -1000, left: -1000 })
+					.prependTo('body').append('<div />').find('div')
+					.css({ width: '100%', height: 200 });
+					scrollbarWidth = 100 - $div.width();
+					$div.parent().remove();
+				}
+			}
+		return scrollbarWidth;
```

Comment:
on this line, you assume that body height is bigger than window height and make calculation according to that.
How about page content already fits the window and there is no scrollbar?
We have to make return line just like that:
return $("body").height() > $(window).height() ? scrollbarWidth : 0;